descriptors/ModuleDescriptor-template.json

```diff
@@ -324,7 +324,8 @@
             "users.item.get",
             "users.collection.get",
             "usergroups.collection.get",
-            "usergroups.item.get"
+            "usergroups.item.get",
+            "inventory-storage.location-units.libraries.collection.get"
```

Comment:
Are libraries the only unit that is fetched, or are campus and institutions also fetched, even if they aren't used?

Reply:
I think only libraries, it's in LocationRepository:78